src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3608,7 +3608,10 @@ namespace ZoneTempPredictorCorrector {
                 if (TempCtrlFound && TempControlledZone(RelativeZoneNum).DeltaTCutSet > 0.0) {
```

Comment:
@lgu1234 When this if is false, `Tprev` is uninitialized below at [line 3630](https://github.com/NREL/EnergyPlus/pull/6754/files#diff-26a720ee34299122c810df4c89b63101R3630).  Looks like the closing } in 3629 should be move down.  Similar problem appears in the SingleCoolingSetPoint block, but DualSetPoint looks OK.

Reply:
@mjwitte You are right. It is a very good catch. Changes are done. Thanks.